### PR TITLE
socket api

### PR DESCRIPTION
**Pull request by CleanBread**
@Avvrik i can't add you as a reviewer, pls check this pr

### CODE REVIEW

File: pages/guides/advanced/live/place-a-bet.mdx [hunk 1]
```diff
@@ -47,6 +47,27 @@ const order: OrderSpec = {
 };
 ```
 
+<Callout type="info">
+To obtain the `minPrise`, utilize the [`calcLivedOdds`](/guides/sdk/utils/calcLiveOdds) function from our SDK in conjunction with the odds data retrieved from the [Socket API](/guides/advanced/live/data-retrieval#get-realtime-markets-updates).
```
Comment (on47sky) [inline pages/guides/advanced/live/place-a-bet.mdx:51]: `minPrice`, `calcLiveOdds`

File: pages/guides/advanced/live/place-a-bet.mdx [hunk 2]
```diff
@@ -47,6 +47,27 @@ const order: OrderSpec = {
...
+
+```ts
+import { calcLivedOdds } from '@azuro-org/sdk'
```
Comment (on47sky) [inline pages/guides/advanced/live/place-a-bet.mdx:54]: there are several typos `calcLivedOdds`, please fix them all :) `calcLiveOdds`

File: pages/guides/advanced/live/data-retrieval.mdx [hunk 3]
```diff
@@ -76,9 +76,78 @@ As for prematch games, live game's markets are constantly updated:
...
+type SocketData = {
+  id: string // condition id
+  state?: ConditionStatus // condition status
+  margin: number
+  reinforcement: number
+  winningOutcomesCount: number
+  outcomes?: Array<{
+    id: number // outcome id
+    price: number // current odds
+    clearPrice: number
```
Comment (on47sky) [inline pages/guides/advanced/live/data-retrieval.mdx:135]: will we add a comment about what a clearPrice is?

Comment (CleanBread) [reply pages/guides/advanced/live/data-retrieval.mdx]: d5bf9ad4162913344b4c0018baddb00da4d291d7

File: pages/guides/advanced/live/place-a-bet.mdx (hunk 1 above)
Comment (CleanBread) [reply pages/guides/advanced/live/place-a-bet.mdx]: d5bf9ad4162913344b4c0018baddb00da4d291d7

**Pull request merged**